webpack/webpack.common.js

```diff
@@ -13,6 +13,8 @@ const aliases = {
     //  remove this file and the yarn modernizr post-install step
     "modernizr": "hqwebapp/js/lib/modernizr",
 
+    "nvd3/nv.d3.latest.min": "nvd3-1.8.6/build/nv.d3.min",
```

Comment:
I suppose this is fine since it looks like dependabot will never upgrade this library (because [package.json depends on the specific version](https://github.com/dimagi/commcare-hq/blob/429a2caf170f0d8187b3bca3f5c50e96d5a5bb5a/package.json#L78)). That doesn't seem like an ideal situation, but I don't have a better suggestion if that's the only way to get the version we need.

Reply:
I wonder if the comment `// version 1.1.10 has a bug that affects line charts with multiple series` should be moved over here?

Reply:
I hadn't thought about it before, but thinking about it now, I think the comment belongs in the app code where it is, since it's explaining why that particular module uses this version instead of the version that the rest of HQ uses.

I think the bigger picture issue here is that nvd3 is effectively EOL, so dependabot isn't going to upgrade it at all. It hasn't been updated [since 2018](https://github.com/novus/nvd3), and I assume it's the reason that we use D3 v3 instead of the current v7.9.